assets/scss/app.scss

```diff
@@ -315,8 +315,8 @@ textarea {
   word-wrap: break-word;
 }
 
-.form-control-100 {
-  width: 100%;
+.form-margin {
+  margin-bottom: 0;
```

Comment:
probably rename this `no-margin` or something. There isn't anything already in hof or govuk?

Reply:
ok will rename, nothing in HOF, it just removes the margin between the textarea and the character count.